.circleci/bin/deploy-sdk.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```

Comment:
the more i think about this i think we should have one way to trigger this. the CI workflow one is actually pretty elegant, I think we should drop this one

Reply:
> we should have one way

`libs/scripts/release.sh` is the script that runs solely within CircleCi's container. This is that "one way".

Additionally:

`triggers/deploy-sdk.py` is the main logic for calling the CircleCi API deployment command.

`bin/deploy-sdk.sh` is a wrapper mainly to ensure dependencies are installed and `$PYTHONPATH` is set correctly.

I strongly vote to keep the command line helper since I know I'll use that more than the UI since I use the command line to grab my commit hashes and I'd like to continue using the command line to run the deployment.

The main goals of automating the SDK Libs release flow were to:

* Always ensure a clean build/deploy environment. ([link](https://audius-internal.slack.com/archives/C03C2V84A1L/p1657843439175639))
* Always have centralized, timestamped logs for what occurred during the build/release process. 

We still reach both goals and provide a UI interface for deployments, for those who chose to use the UI.

> CI workflow one is actually pretty elegant

I find running the command line utility much more elegant than: 

* Finding my git commit
* Visiting CircleCi
* Perform 4 mouse clicks on a janky UI
* Waiting for the job to load in the UI to be able to monitor the logs

If we don't get much use out of the command line helper in the next sprint, I'd be open to removing it. That allows @sliptype and his team time to see if the command line helper is more useful over the course of a few releases.

Thoughts?

Reply:
im open to having a script. i was just thinking one way of doing it like just ci would be easier to teach the team and also debug in case of problems but i'm open to having both of these